integration-test/src/test/scala/com/cloudera/livy/test/JobApiIT.scala

```diff
@@ -157,16 +157,17 @@ class JobApiIT extends BaseIntegrationTestSuite with BeforeAndAfterAll with Logg
     assert(result === "hello")
   }
 
-  test("run scala jobs") {
+  test("run echo jobs") {
```

Comment:
That really increase the pom modules, I thought your suggestion before (build two versions), but just for one test may not be so necessary. Also I'm not sure what actually tested by using Scala code + Java API client.